### ISSUE TITLE
will it be possible in futur version...

### ISSUE THREAD

**Issue by mxmkmarquette** [alexmacarthur/wp-better-resource-hints#4]
to add something that maybe auto detect the script and else, than something that you have to add like every script from your website manually.

will be a great addition to this mega ultra super plugin.

**Comment by alexmacarthur** [comment]
Thanks for the suggestion! Are you referring to prefetching specifically? I have been toying with a better way to implement that, and would like to have a maintainable and reliable way knock that out soon. Until then, manually designating assets is the safest way to go.

**Comment by mxmkmarquette** [comment]
yeah because the other we have option but Prefetching could be a nice addition to have either some option or something that detect the thingy, but if you have something in mind will wait and see cause i find this plugin really usefull :)
